ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -89,6 +90,11 @@ public MergeVisitsPage clickOnMergeVisits() {
         return new MergeVisitsPage(this);
     }
 
+    public ClinicianFacingPatientDashboardPage goToPatientDashboard() {
+        clickOn(RETURN_TO_DASHBOARD);
```

Comment:
You can refactor https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/page/StylesGuidePage.java#L14 into the parent class and use that there and here